evap/staff/templates/staff_semester_preparation_reminder.html

```diff
@@ -61,3 +71,27 @@
     {% endfor %}
 
 {% endblock %}
+
+{% block modals %}
+    {{ block.super }}
+    {% trans 'Remind all' as title %}
+    {% trans 'Do you really want to remind everyone?' as question %}
```

Comment:
In the issue (#1081), @janno42 wrote that the list of responsibles that will be reminded should be shown. I like that idea. Putting together the list here is probably a bit annoying with django's template language, so I'd suggest we build up the question text in the view. What do you think?

Reply:
I think it's fine if we don't have the list because it would be too much work for very little benefit.